content/en/blog/2019/v1beta1-authorization-policy/index.md

```diff
@@ -0,0 +1,464 @@
+1. Repeat the process for the next service enabled with RBAC.
+
+For each namespace enabled with RBAC:
+
+1. Apply a `v1beta1` policy that denies all traffic to the given namespace.
+
+### Migration Example
+
+Assume you have the following `v1alpha1` policies for the `httpbin` service in the `foo` namespace:
+
+{{< text yaml >}}
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ClusterRbacConfig
+metadata:
+  name: default
+spec:
+  mode: 'ON_WITH_INCLUSION'
+  inclusion:
+    namespaces: ["foo"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: httpbin
+  namespace: foo
+spec:
+  rules:
+  - services: ["httpbin.foo.svc.cluster.local"]
+    methods: ["GET"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRoleBinding
+metadata:
+  name: httpbin
+  namespace: foo
+spec:
+  subjects:
+  - user: "cluster.local/ns/default/sa/sleep"
+  roleRef:
+    kind: ServiceRole
+    name: "httpbin"
+{{< /text >}}
+
+You migrate the above policies to `v1beta1` in the following ways:
+
+1. Assume the `httpbin` service has the following workload selector:
+
+    {{< text yaml >}}
+    selector:
+      app: httpbin
+      version: v1
+    {{< /text >}}
+
+1. Create a `v1beta1` policy with the workload selector:
+
+    {{< text yaml >}}
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+     name: httpbin
+     namespace: foo
+    spec:
+     selector:
+       matchLabels:
+         app: httpbin
+         version: v1
+    {{< /text >}}
+
+1. Update the `v1beta1` policy with each `ServiceRole` and `ServiceRoleBinding`
+applied to the service:
+
+    {{< text yaml >}}
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+     name: httpbin
+     namespace: foo
+    spec:
+     selector:
+       matchLabels:
+         app: httpbin
+         version: v1
+     rules:
+     - from:
+       - source:
+           principals: ["cluster.local/ns/default/sa/sleep"]
+       to:
+       - operation:
+           methods: ["GET"]
+    {{< /text >}}
+
+1. Apply the `v1beta1` policy and monitor the traffic to make sure it works as expected
+
+1. Apply the following `v1beta1` policy that denies all traffic to the `foo` namespace
+because the `foo` namespace is enabled with RBAC:
+
+    {{< text yaml >}}
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+     name: deny-all
+     namespace: foo
+    spec:
+     {}
+    {{< /text >}}
+
+Make sure the `v1beta1` policy is working as expected and then you can delete the `v1alpha1` policies
+from the cluster.
+
+### Automation of the Migration
+
+To help ease the migration, we're working on a tool `istioctl x authz convert` to automate the
+conversion. The tool is still under active development and doesn't support all `v1alpha1` semantics
+so we recommend to do the migration manually in Istio 1.4.
+
+The tool will be ready for use in Istio 1.5 and will be able to fetch all the
```

Comment:
As discussed, we can release it in a minor version 1.4.x. And always give warning if it cannot convert an alpha policy.

Reply:
Updated for the support in the patch release.